lib/src/utils/mod.rs

```diff
@@ -65,6 +65,7 @@ pub struct Media {
 
     #[serde(skip_serializing, skip_deserializing)]
     pub index: Option<usize>,
+    pub title: String,
```

Comment:
Here also a `Option<String>` and on top a: `#[serde(skip_serializing_if = "Option::is_none")]`